src/api-specification/contribution-controller/retrieving-contributions.md

```diff
@@ -0,0 +1,136 @@
+| `org`              | `array[string]`   | The country of origination of the fraud event. Set as a two-letter country code (Alpha-2, ISO 3166; e.g., US, GA).                                                                                                                                                                                                                                                                                                               |
+| `self-only`        | `boolean`         | A boolean that defines whether the response should only contain contributions submitted by the requesting peer. <br> The `self-only` parameter is set to `false` by default.                                                                                                                                                                                                                                                     |
+| `fetch-mode`       | `string`          | The parameter that filters the response entries based on whether the requesting user has already seen them. <br> Can be one of the following: <ol><li>`NEW` — includes only the unseen by the requesting user entries.</li><li>`OLD` — includes only the already seen by the requesting user entries.</li><li>`DEFAULT` — includes all of the entries.</li></ol> <br> The `fetch-mode` parameter is set to `DEFAULT` by default. |
+| `confidence-score` | `boolean`         | A boolean that defines whether the price of the retrieved contributions is affected by the [confidence index](../../overview/tokenomics.md#confidence-index). If set to `true`, the price of any given contribution is expected to rise above the [default conversion rate](../../overview/tokenomics.md#current-conversion-rate). <br> The `confidence-score` parameter is set to `false` by default.                   |
+
+#### Input/request structure
+
+A `GET` request to the endpoint.
+
+#### Output/response structure
+
+::: details Show
+
+::: code-group Data structure
+
+```json5 [Structure]
+{
+  "status": {
+    "code": integer($int32),
+    "name": "string",
+    "message": "string"
+  },
+  "data": {
+    "contributions": [
+        {
+          "id": "string", //127.0.0.1 OR 127.0.0.1-127.0.0.2 OR +14155552671 OR +14155552671-+14155552672 OR 107615702016566
+          "fraudType": "string",
+          "origination": "string",
+          "destination": "string",
+          "expiryDate": integer($int32),
+          "fraudStatus": "string(enum)", //'ACTIVE' OR 'EXPIRED' OR 'FLAGGED'
+          "confidenceIndex": number($double),
+          "isPrivileged": boolean,
+          "peerId": "string",
+          "flagger": "string",
+          "timestamp": integer($int32)
+        }
+    ],
+    "details": {
```

Comment:
I guess it deserves more elaboration (refer to Figma states of this operation).
There can be:
* Full retrieval (enough credits)
* Partial retrieval (not enough credits)
* Empty retrieval (not enough credits)
* Empty retrieval (no data for the filters specified)